lld/ELF/Relocations.cpp

```diff
@@ -2031,17 +2031,44 @@ void ThunkCreator::mergeThunks(ArrayRef<OutputSection *> outputSections) {
       });
 }
 
-static int64_t getPCBias(Ctx &ctx, RelType type) {
-  if (ctx.arg.emachine != EM_ARM)
-    return 0;
-  switch (type) {
-  case R_ARM_THM_JUMP19:
-  case R_ARM_THM_JUMP24:
-  case R_ARM_THM_CALL:
-    return 4;
-  default:
-    return 8;
+static const uint32_t HEXAGON_MASK_END_PACKET = 3 << 14;
```

Comment:
constexpr

namespace-scope const variables are of inline linkage by default, so you can omit `static`

Reply:
fixed, thank you